apps/geo/dataloaders.py

```diff
@@ -32,6 +32,15 @@ def batch_load_fn(self, keys):
         return Promise.resolve([_map.get(key) for key in keys])
 
 
+class GeoAreaLoader(DataLoaderWithContext):
+    def batch_load_fn(self, keys):
+        geo_area_qs = GeoArea.objects.filter(id__in=keys).defer('polygons', 'centroid', 'cached_data')
+        _map = defaultdict()
```

Comment:
defaultdict is used for nested structure. For flat, basic dict is fine.
```suggestion
        _map = {}
```